sql/core/src/main/scala/org/apache/spark/sql/execution/ui/ExecutionPage.scala

```diff
@@ -148,6 +155,8 @@ class ExecutionPage(parent: SQLTab) extends WebUIPage("execution") with Logging
   }
 
   private def modifiedConfigs(modifiedConfigs: Map[String, String]): Seq[Node] = {
+    if (modifiedConfigs.isEmpty) return Nil
```

Comment:
it's nice, acutally save the life. I always click SQL Properties even if the Spark version is old ...

But it can be null that we should handle it carefully

Reply:
oh, yeah. will handle it. Thanks!